### PR TITLE
core: frontend: Implement motor test 

### PR DESCRIPTION
**Pull request by Williangalvani**
needs some cleaning up, and probably refactoring the the arm button into a component

tested with a servo in the pwm rail and 5v jumped to it

### CODE REVIEW

**Comment by Williangalvani** [comment]
change color of main bhar
make body transaprent in pwm outputs
look at cockpit's armed button

**Comment by Williangalvani** [comment]
we still need to able to revert the motor direction, along with a couple of other things. I intend to do this as it's own component in another pr

**Review by patrickelectric — CHANGES_REQUESTED** [review]

File: core/frontend/src/store/mavlink.ts [hunk 1]
```diff
@@ -47,6 +47,9 @@ class MavlinkStore extends VuexModule {
 
     // Create a new listener
     this.message_listeners[messageName] = mavlink2rest.startListening(messageName).setCallback((receivedMessage) => {
+      if (receivedMessage.header.component_id !== 1) {
```
[the remark this answers is not in the document]
Comment (Williangalvani) [reply core/frontend/src/store/mavlink.ts:50]: removed it. I'm not sure of why it was there in the first place 🤔

File: core/frontend/src/components/vehiclesetup/PwmSetup.vue [hunk 2]
```diff
@@ -131,13 +172,48 @@ export default Vue.extend({
...
+    this.desired_armed_state = this.is_armed
+  },
+  beforeDestroy() {
+    clearInterval(this.motor_zeroer_interval)
+    clearInterval(this.motor_writer_interval)
+    mavlink.setMessageRefreshRate({ messageName: 'SERVO_OUTPUT_RAW', refreshRate: 1 })
   },
   methods: {
+    styleForMotorBar(value: number): string {
+      const percent = (value - 1500) / 10
```
[the remark this answers is not in the document]
Comment (Williangalvani) [reply core/frontend/src/components/vehiclesetup/PwmSetup.vue:213]: motors don't use SERVOn_MIN/MAX

We can expose MOT_PWM_MIN/MAX in another slider, I think.

[the remark this answers is not in the document]
Comment (Williangalvani) [reply core/frontend/src/components/vehiclesetup/PwmSetup.vue:213]: we don't really have TRIM params for motors, though 😅

**Comment by patrickelectric** [comment]
@Williangalvani can you rebase over master ?

File: core/frontend/src/components/vehiclesetup/PwmSetup.vue [hunk 3]
```diff
@@ -146,6 +222,133 @@ export default Vue.extend({
...
+            param2: force ? 21196 : 0, // force arming/disarming (override preflight checks and disarming in flight)
+            param3: 0,
+            param4: 0,
+            param5: 0,
+            param6: 0,
+            param7: 0,
+            command: {
+              type: MavCmd.MAV_CMD_COMPONENT_ARM_DISARM,
+            },
+            target_system: 1,
```
Comment (patrickelectric) [inline core/frontend/src/components/vehiclesetup/PwmSetup.vue:280]: That will be affect by #1581, do you prefer to merge it before ?

File: core/frontend/src/components/vehiclesetup/PwmSetup.vue [hunk 4]
```diff
@@ -146,6 +222,133 @@ export default Vue.extend({
...
+.pwm-bar {
+    /* eslint-disable-next-line */
+    background-image: linear-gradient(135deg, rgba(255, 255, 255, 0.25) 25%, transparent 0, transparent 50%,
+      rgba(255, 255, 255, 0.25) 0, rgba(255, 255, 255, 0.25) 75%, transparent 0, transparent);
+    background-size: 40px 40px;
+    background-repeat: repeat;
+    height: inherit;
+    left: 0;
+    position: absolute;
+    transition: none;
```
Comment (patrickelectric) [inline core/frontend/src/components/vehiclesetup/PwmSetup.vue:338]: the indentation of this css does not match with the others

**Review by patrickelectric — APPROVED** [review]

**Pull request merged**